tsconfig.es.json

```diff
@@ -2,6 +2,7 @@
   "extends": "./tsconfig.json",
   "compilerOptions": {
     "module": "es6",
-    "outDir": "es"
+    "outDir": "es",
+    "types": ["express"]
```

Comment:
I tried this but it doesn't seem to stop "node" from being visible in compilation.